llvm/test/CodeGen/AMDGPU/fneg-combines.ll

```diff
@@ -41,6 +42,57 @@ define amdgpu_kernel void @v_fneg_add_f32(ptr addrspace(1) %out, ptr addrspace(1
 ; GCN-NEXT: flat_store_dword v{{\[[0-9]+:[0-9]+\]}}, [[ADD]]
 ; GCN-NEXT: s_waitcnt vmcnt(0)
 define amdgpu_kernel void @v_fneg_add_store_use_add_f32(ptr addrspace(1) %out, ptr addrspace(1) %a.ptr, ptr addrspace(1) %b.ptr) #0 {
```

Comment:
Does this test pass? Looks like you should have deleted the manual checks above this line, now that there are autogenerated checks below this line.

Reply:
This isn't supposed to be here